docs/contributing/forms.md

```diff
@@ -120,7 +124,8 @@ Add custom validation with the `customValidator` property.
 A custom validator is an object with a `validate` method that returns a `Validation` object.
 
 ```js
-import { Validation } from '@clevercloud/components/lib/form/validation.js';
+// this path assumes you are importing it a from a component file in `src/components/cc-component-name/cc-component-name.js`
+import { Validation } from '../lib/form/validation.js';
```

Comment:
nitpick: the relative import will be from `../../` and not `../` :eyes:

Reply:
:sweat_smile: thanks!